framework/src/problems/DumpObjectsProblem.C

```diff
@@ -35,6 +35,9 @@ DumpObjectsProblem::DumpObjectsProblem(const InputParameters & parameters)
   _nl = _nl_sys;
   _aux = std::make_shared<AuxiliarySystem>(*this, "aux0");
   newAssemblyArray(*_nl_sys);
+
+  // Create etra vectors and matrices if any
```

Comment:
`etra` -> `extra`.

And the `CreateTagVectors()` call will have to be fixed.

There are 3 other occurrences of this, so they need to be fixed as well.